### ISSUE TITLE
ZwifterBikes links

### ISSUE THREAD

**Issue by andipaetzold** [andipaetzold/zwiftmap#1316]
- [ ] Segments
- [ ] Routes

https://github.com/andipaetzold/zwift-data/issues/26

**Comment by sandraqu** [comment]
@andipaetzold Would you give more context to the tasks above and also, to the mappings?  I notice the mappings have a few missing.  Do the missing ones not exist?  Thanks!

**Comment by andipaetzold** [comment]
Hey @sandraqu 👋 

ZwifterBike urls were added to `zwift-data` in `1.28.0`. The developer of the website sent me the mapping file attached in the `zwift-data` ticket.

I think I just forgot to close both tasks. ZwifterBike urls are already rendered on zwiftmap.com

New routes released after January 2023 likely don't have the ZwifterBike urls set. Reason being that I updated `zwift-data` before ZwifterBikes updated its page, so the links didn't exist at that point. I never checked back on the page after that.

There are probably quite a bit of routes that have ZwifterBike urls but the data is missing in zwift-data. Checking would need to happen manually, route by route, and I didn't have time/motivation to do that, yet.
